CHANGES.md

```diff
@@ -57,6 +57,7 @@
 * New highly anticipated feature X added to Python SDK ([#X](https://github.com/apache/beam/issues/X)).
 * New highly anticipated feature Y added to Java SDK ([#Y](https://github.com/apache/beam/issues/Y)).
```

Comment:
Since we're close to the cut, while we're here can you remove these highlight pieces? (same with highlights/deprecations/bug fixes below)

Reply:
Can do.